src/Post/postsService.js

```diff
@@ -1,8 +1,14 @@
+import AuthUser from "../User/Auth/AuthUser.js"
 import Post from "./Post.js"
 
 class PostsService {
+    async getAll() {
```

Comment:
Could you use then everywhere please instead of async/await?

Reply:
The same for try-catch

Reply:
used then()